pt-BR/active_record_migrations.md

```diff
@@ -4,36 +4,26 @@
 Active Record Migrations
 ========================
 
-Migrations are a feature of Active Record that allows you to evolve your
-database schema over time. Rather than write schema modifications in pure SQL,
-migrations allow you to use an easy Ruby DSL to describe changes to your
-tables.
+*Migrations* são uma funcionalidade do *Active Record* que permite expandir nosso esquema do banco de dados com o tempo. Ao invés de escrever modificações no esquema em SQL puro,*migrations* permitem o uso de uma *Ruby DSL* simples para descrever as mudanças em nossas tabelas.
 
-After reading this guide, you will know:
+Depois de ler esse guia, você entenderá:
 
-* The generators you can use to create them.
-* The methods Active Record provides to manipulate your database.
-* The rails commands that manipulate migrations and your schema.
-* How migrations relate to `schema.rb`.
+* Os *generators* que você pode usar para criar as *migrations*.
+* Os métodos que o *Active Record* provê para manipular seu banco de dados.
+* Os comandos *rails* que manipulam *migrations* e o esquema do seu banco.
```

Comment:
```suggestion
* Os comandos `rails` que manipulam *migrations* e o esquema do seu banco.
```

To na dúvida @alemlima e @campuscode/guia-rails-team 

Mas aqui parece que eles estão referenciando o termo rails usado nas rakes de migration então imaginei colocar como código. O que acham?

Reply:
Acho que é isso mesmo Henrique, assim melhor mesmo.